### DISCUSSION TITLE
How to deploy the production Gatsby site.

### DISCUSSION THREAD

**Discussion by dgallardox** [gatsbyjs/gatsby#36639] [Help]
So I understand that on ```npm run build``` a /public folder is created with the static version of the site. My hosting provider runs ```npm run build``` and then ```npm run start``` ( which = gatsby develop ) to deploy the site. What I am wondering is that since ```npm run start``` is used to deploy the site. Is the production version even being served? Thanks!

<img width="767" alt="Screen Shot 2022-09-19 at 12 41 59 PM" src="https://user-images.githubusercontent.com/101141475/191079905-8099a7d9-901b-4681-8760-25ea99cb6279.png">

**Comment by PaulieScanlon** [comment]
Hi @dgallardox take a look at this page from our docs. There are a number of links to help you with deploying.

https://www.gatsbyjs.com/docs/deploying-and-hosting/

npm run start / gatsby develop is used to start the development server. This isn't required when serving the built site in production.

**Reply by dgallardox** [reply]
So my site is deployed from my root folder instead of the ./public , does this mean the non-static site is being served? I guess I am confused as to how you tell the hosting provider to deploy the site from /public

**Reply by PaulieScanlon** [reply]
In the hosting provider config you'll need to set /public as the root directory. Which hosting provider are you using, is there a guide listed on our docs? 

If not, perhaps it's one we can consider adding.

**Reply by dgallardox** [reply]
I am using WP Engine and cannot change the root dir to /public. I was able to get it working with: https://www.gatsbyjs.com/plugins/gatsby-plugin-fastify/

I think getting a guide on getting it working with WP Engine would be awesome! Am I able to contribute to the docs?
